docs/README.md

```diff
@@ -156,26 +138,108 @@ forc index check
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | fuel-indexer binary    |  /Users/rashad/.fuelup/bin/fuel-indexer                 |
++--------+------------------------+---------------------------------------------------------+
+|   ⛔️   | fuel-indexer service   |  Failed to detect service at Port(29987).               |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | psql                   |  /usr/local/bin/psql                                    |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | fuel-core              |  /Users/rashad/.fuelup/bin/fuel-core                    |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | docker                 |  /usr/local/bin/docker                                  |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | fuelup                 |  /Users/rashad/.fuelup/bin/fuelup                       |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | wasm-snip              |  /Users/rashad/.cargo/bin/wasm-snip                     |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | forc-postgres          |  /Users/rashad/.fuelup/bin/fuelup                       |
++--------+------------------------+---------------------------------------------------------+
+```
+
+### 2.2 Database setup
+
+To quickly setup and bootstrap the PostgreSQL database that we'll need, we'll use the `forc-postgres` plugin that is included in `fuelup`.
+
+> IMPORTANT: Ensure that any local PostgreSQL instance that is running on port `5432` is stoppe
+
+```bash
+forc index postgres create postgres --persistent
+```
+
+
+```text
+Downloading, unpacking, and bootstrapping database.
+▸▹▹▹▹ ⏱  Setting up database...                                                                                          🎬 Starting database at 'postgres://postgres:postgres@localhost:5432/postgres'.
+💡 Creating database at 'postgres://postgres:postgres@localhost:5432/postgres'.
+waiting for server to start....2023-02-09 16:11:25.764 EST [86849] LOG:  starting PostgreSQL 14.6 on x86_64-apple-darwin20.6.0, compiled by Apple clang version 12.0.0 (clang-1200.0.32.29), 64-bit
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on IPv6 address "::1", port 5432
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on IPv4 address "127.0.0.1", port 5432
+2023-02-09 16:11:25.766 EST [86849] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
+2023-02-09 16:11:25.769 EST [86850] LOG:  database system was shut down at 2023-02-09 16:06:37 EST
+2023-02-09 16:11:25.772 EST [86849] LOG:  database system is ready to accept connections
```

Comment:
Can we remove some of the lower level logs from the `forc postgres` blurb? I think some of it is valuable but other parts (e.g. "received fast shutdown request") don't seem necessary and make the blocks longer.

Reply:
@deekerno Ah, looks like the output could be formatted not-great (thus the long lines). Will fix